management/src/clusterm/manager/client_test.go

```diff
@@ -89,53 +100,59 @@ func (s *managerSuite) TestPostSuccess(c *C) {
 		url: baseURL,
 	}
 
+	var reqHostGroupBody bytes.Buffer
+	c.Assert(json.NewEncoder(&reqHostGroupBody).Encode(testReq1HostGroupBody), IsNil)
+
+	var testFlags ActionFlags
 	tests := map[string]struct {
 		expURLStr string
 		nodeName  string
-		extraVars string
+		flags     ActionFlags
 		exptdBody []byte
-		cb        func(name, extraVars string) error
+		cb        func(name string, flags ActionFlags) error
 	}{
 		"commission": {
 			expURLStr: fmt.Sprintf("http://%s/%s/%s", baseURL, PostNodeCommissionPrefix, testNodeName),
 			nodeName:  testNodeName,
-			extraVars: "",
-			exptdBody: []byte{},
+			flags:     ActionFlags{"", ansibleMasterGroupName},
+			exptdBody: reqHostGroupBody.Bytes(),
```

Comment:
@mapuri not sure if `okReturner` and `failureReturner` would need a change. Both of them do a deepEquals on `r.Body`.. So as long as we can pass the correct `exptBody` - which has the hostGroup populated. The existing test should take care of itself. I have tested it.  both positive and negative scenario to see that the logic works.